libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -360,6 +422,82 @@ void HarmonicNotchFilter<T>::reset()
     }
 }
 
+#if HAL_LOGGING_ENABLED
+// @LoggerMessage: FCN
+// @Description: Filter Center Message - per motor
+// @Field: TimeUS: microseconds since system startup
+// @Field: I: instance
+// @Field: NDn: number of active dynamic harmonic notches
+// @Field: CF1: centre frequency for motor 1
+// @Field: CF2: centre frequency for motor 2
+// @Field: CF3: centre frequency for motor 3
+// @Field: CF4: centre frequency for motor 4
+// @Field: CF5: centre frequency for motor 5
```

Comment:
I know you have only gone to 5 here to take account of quadplanes, but its a bit weird for copters and artificially excludes hexes. I suggest we go to 6 giving the same number of fields as in FTN

Reply:
changed